get_started_with_membrane/06_observability_and_logging.md

```diff
@@ -0,0 +1,56 @@
+# Observability & logging
+
+To observe and debug Membrane, you can use a variety of tools provided by the BEAM, like [observer](https://www.erlang.org/doc/man/observer.html) or [Process.info](https://hexdocs.pm/elixir/1.15.5/Process.html#info/1). Apart from them, Membrane provides dedicated utilities for observability and logging.
+
+## Introspecting pipelines with kino_membrane
+
+[kino_membrane](https://github.com/membraneframework/kino_membrane) allows for introspecting Membrane pipelines in [Livebook](https://livebook.dev). You can either run the pipeline inside livebook ([see example](https://hexdocs.pm/kino_membrane/pipeline_in_livebook.html)) or connect to a running VM ([see example](https://hexdocs.pm/kino_membrane/connect_to_node.html)). Then, it's enough to install kino_membrane:
+
+```elixir
+Mix.install([:kino_membrane])
+```
+
+and run
+
+```elixir
+KinoMembrane.pipeline_dashboard(pipeline)
+```
+
+in Livebook, and you'll get a panel with the pipeline graph, metrics and other utilities:
+
+![Kino Membrane](assets/images/kino_membrane.png)
+
+Check [kino_membrane](https://github.com/membraneframework/kino_membrane) for details.
+
+## Logging
+
+Apart from the usual `Logger` configuration, logging in Membrane can be additionally configured, also via Elixir's `Config`. It allows to enable verbose mode and customize metadata, for example:
+
+```elixir
+import Config
+config :membrane_core, :logger, verbose: true
```

Comment:
I would notice, that it is necessary to recompile modules after changing this config

Reply:
done